src/symbols.cs

```diff
@@ -0,0 +1,215 @@
+using CoreFoundation;
+using ObjCRuntime;
+using Foundation;
+
+using System;
+
+namespace Symbols {
+	[Watch (10, 0), TV (17, 0), Mac (14, 0), iOS (17, 0), MacCatalyst (17, 0)]
+	[BaseType (typeof (NSObject))]
+	interface NSSymbolEffect : NSCopying, NSSecureCoding { }
```

Comment:
Documentation says it should not be used.

```suggestion
	[BaseType (typeof (NSObject))]
	[DisableDefaultCtor]
	interface NSSymbolEffect : NSCopying, NSSecureCoding { }
```